src/main_app/jobs_workers/admin_jobs_workers/update_owid_charts/objects.py

```
@@ -0,0 +1,54 @@
+        "failed": 0,
+    },
+    "charts_processed": [],
+    "updated_charts": [],
+    "skipped_charts": [],
+    "failed_charts": [],
+}
+
+__all__ = [
+    "UpdateOwidChartsWorkerObject",
+    "UpdateOwidChartsSummary",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)